client/src/partials/journal_voucher/journal_voucher.js

```diff
@@ -63,9 +63,11 @@ angular.module('bhima.controllers')
         'tables' : {
```

Comment:
This query does not allow user to add a transaction for the supplier, only for the employee.
Please correct it.